### PR TITLE
Add discovered host summary to dashboard

### PR DESCRIPTION
**Pull request by GregSutcliffe**
@abenari, how does this look to you?

### CODE REVIEW

File: app/overrides/add_dashboard_table.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Deface::Override.new(:virtual_path => "dashboard/_status_table",
+                     :name => "add_dashboard_table",
+                     :insert_after => "ul:last",
+                   :text => '
+<h4 class="header"><%= "Discovery Status" -%></h4>
+<ul>
+  <li>
+    <%= link_to("Discovered Hosts", discovers_path, :class=>"dashboard-links" ) %>
+    <h4><%= Host::Discovered.all.count %></h4>
```
Comment (ohadlevy) [inline app/overrides/add_dashboard_table.rb]: all.count is not required, just count would perform better (no need to convert to array first)

**Comment by GregSutcliffe** [comment]
@ohadlevy good catch, updated to just use .count

**Comment by GregSutcliffe** [comment]
@abenari ping? Does this seem ok to you?

File: app/overrides/add_dashboard_table.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Deface::Override.new(:virtual_path => "dashboard/_status_table",
+                     :name => "add_dashboard_table",
+                     :insert_after => "ul:last",
+                   :text => '
+<h4 class="header"><%= "Discovery Status" -%></h4>
```
Comment (abenari) [inline app/overrides/add_dashboard_table.rb]: Can we translate the header?  _("Discovery Status")

File: app/overrides/add_dashboard_table.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+<ul>
+  <li>
+    <%= link_to("Discovered Hosts", discovers_path, :class=>"dashboard-links" ) %>
```
Comment (abenari) [inline app/overrides/add_dashboard_table.rb]: Same as above translation.

**Comment by abenari** [comment]
Looks good to me, fits well in the page.

I was thinking of the discovered hosts as a host pool, with unassigned hosts. We could have 'host pool' in the title and 'unassigned hosts' in the link, what do you think?

File: app/overrides/add_dashboard_table.rb (hunk 2 above)
Comment (GregSutcliffe) [inline app/overrides/add_dashboard_table.rb]: I don't think translations work in plugins yet - we need to figure out how to get the strings into Transifex. See https://trello.com/card/as-a-foreman-user-i-would-like-plugins-to-be-localized/5162af6bb868365e0c007135/68

File: app/overrides/add_dashboard_table.rb (hunk 3 above)
Comment (GregSutcliffe) [inline app/overrides/add_dashboard_table.rb]: See above comment. I agree, I'd like it to be localized ;)

**Comment by GregSutcliffe** [comment]
Merged as 890ef2a. Thanks for the review, guys.

**Pull request closed without merging**